### PR TITLE
JPEG: ICC parsing, display of ICC color space

### PR DESCRIPTION
**Pull request by JeromeMartinez**
```
<track type="Image">
<ColorSpace>YUV</ColorSpace>
<extra>
<ColorSpace_ICC>RGB</ColorSpace_ICC>
</extra>
</track>
```

### CODE REVIEW

File: Source/MediaInfo/Image/File_Jpeg.cpp [hunk 1]
```diff
@@ -207,6 +207,43 @@ string Jpeg2000_Rsiz(int16u Rsiz)
...
+        case 0x434D594B: return "CMYK";
+        case 0x47524159: return "Y";
+        case 0x484C5320: return "HLS";
+        case 0x48535620: return "HSV";
+        case 0x4C616220: return "Lab";
+        case 0x4C757620: return "Luv";
+        case 0x52474220: return "RGB";
+        case 0x58595A20: return "XYZ";
+        case 0x59436272: return "YCbCr";
+        case 0x59787920: return "xyY";
```
Comment (dericed) [inline Source/MediaInfo/Image/File_Jpeg.cpp]: Yxy?

File: Source/MediaInfo/Image/File_Jpeg.cpp [hunk 2]
```diff
@@ -207,6 +207,43 @@ string Jpeg2000_Rsiz(int16u Rsiz)
...
+    }
+}
+
+string ICC_ColorSpace(int32u ColorSpace)
+{
+    switch (ColorSpace)
+    {
+        case 0x434D5920: return "CMY";
+        case 0x434D594B: return "CMYK";
+        case 0x47524159: return "Y";
```
Comment (dericed) [inline Source/MediaInfo/Image/File_Jpeg.cpp]: GRAY?

File: Source/MediaInfo/Image/File_Jpeg.cpp [hunk 3]
```diff
@@ -1189,6 +1226,196 @@ void File_Jpeg::APP1_EXIF()
...
+            #endif //MEDIAINFO_TRACE
+        Element_End0();
+        Skip_C4(                                                "'acsp' profile file signature ");
+        Skip_C4(                                                "Primary platform signature");
+        Skip_B4(                                                "Profile flags");
+        Skip_C4(                                                "Device manufacturer");
+        Skip_B4(                                                "Device model");
+        Skip_B4(                                                "Device attributes TODO 1");
+        Skip_B4(                                                "Device attributes TODO 2");
+        Skip_B4(                                                "Rendering Intent ");
```
Comment (dericed) [inline Source/MediaInfo/Image/File_Jpeg.cpp]: trailing space

File: Source/MediaInfo/Image/File_Jpeg.cpp [hunk 4]
```diff
@@ -1189,6 +1226,196 @@ void File_Jpeg::APP1_EXIF()
...
+        Element_Begin1("nCIEXYZ values");
```
Comment (dericed) [inline Source/MediaInfo/Image/File_Jpeg.cpp]: from http://www.color.org/icc1V42.pdf "XYZ values" would be better

File: Source/MediaInfo/Image/File_Jpeg.cpp (hunk 1 above)
Comment (JeromeMartinez) [reply Source/MediaInfo/Image/File_Jpeg.cpp]: Fro ICC spec it is Yxy, but I see xyY at https://en.wikipedia.org/wiki/CIE_1931_color_space#CIE_xy_chromaticity_diagram_and_the_CIE_xyY_color_space
which one is the more classic?

File: Source/MediaInfo/Image/File_Jpeg.cpp (hunk 2 above)
Comment (JeromeMartinez) [reply Source/MediaInfo/Image/File_Jpeg.cpp]: MediaInfo uses "Y" for gray in other parsers, using 1 letter per component (make things easier when e.g. you add "A" to this line).

File: Source/MediaInfo/Image/File_Jpeg.cpp (hunk 4 above)
Comment (JeromeMartinez) [reply Source/MediaInfo/Image/File_Jpeg.cpp]: Spec say "The nCIEXYZ values of the illuminant of the PCS"
Both look wrong :).
Will change to "Illuminant of the PCS"

Comment (JeromeMartinez) [reply Source/MediaInfo/Image/File_Jpeg.cpp]: " from http://www.color.org/icc1V42.pdf ": it is [deprecated by v4.3](http://www.color.org/icc_specs2.xalter)

**Pull request merged**